os/arch/arm/src/armv7-a/arm_cpuhead.S

```diff
@@ -244,9 +245,11 @@ __cpu3_start:
 	mov		r2, #PGTABLE_SIZE
 	mla		r1, r2, r5, r1		/* page table of cpu1,2,3 */
 #endif
-	orr		r1, r1, #(TTBR0_RGN_WBWA | TTBR0_IRGN0)	/* Select cache properties */
+	orr		r1, r1, #(TTBR0_RGN_WBWA | TTBR0_IRGN0 | TTBR0_S)	/* Select cache properties */
 	mcr		CP15_TTBR0(r1)
+#ifndef CONFIG_ARCH_CORTEXA32
 	mcr		CP15_TTBR1(r1)
+#endif
```

Comment:
Why is the change necessary? We will be only using ttbr0 so ttbr1 can be set regardless.

Reply:
I think that since we are not utilizing ttbr1, it is meaningless to set it. If you think this change is redundant, I can remove it.

Reply:
Yes, please revert it if it doesn't cause any issue. Lets only necessary changes under config checks.

Reply:
Hello @abhishek-samsung ,

Okay, I will make changes to all the comment once power save PR is merged. As there is dependency in between these two, a rebase is needed for SMP. In the meantime, you may review and leave comment based on your concern.